wdl-lint/tests/lints/missing-runtime-block/source.errors

```diff
@@ -6,3 +6,12 @@ warning[MissingRuntime]: task `bad` is missing a `runtime` section
   │
   = fix: add a `runtime` section to the task
 
+note[RuntimeSectionKeys]: the following runtime key is recommended by the WDL v1.1 specification: `container`
```

Comment:
`#@ except` this?

Reply:
This case is very similar to https://github.com/stjude-rust-labs/wdl/issues/116, where "fixing" one rule immediately fires off a related lint. We should be smarter about how we handle this, but I can't think of any fixes off the top of my head. IMO these are closely enough related that the diagnostic should be left.